dttools/src/link.c

```diff
@@ -532,30 +532,17 @@ struct link *link_serve_address(const char *addr, int port)
 int link_ssl_wrap_server(struct link *link, const char *key, const char *cert) {
 #ifdef HAS_OPENSSL
 	if(key && cert) {
-		link->ctx = _create_ssl_context(/* is client */ 0);
-		_set_ssl_keys(link->ctx, key, cert);
-
-		return 1;
-	}
-#endif
-
-	return 0;
-}
-
-
-int link_ssl_wrap_accept(struct link *parent, struct link *link) {
-#ifdef HAS_OPENSSL
-	if(parent->ctx) {
 		debug(D_TCP, "setting up ssl state for %s port %d", link->raddr, link->rport);
 
-		link->ctx = _create_ssl_context(/* is client */ 1);
```

Comment:
Drop the argument from `_create_ssl_context`, and use `TLS_method()` or `SSLv23_method()` as appropriate.